README.md

```diff
@@ -27,12 +29,18 @@ put the gpg_key_id into vars.yaml and then run setup
 # building releases (to be automated later)
 
 ## building for arm64
-`cd /srv/src/peach-oled
+```cd /srv/src/peach-oled
 cargo-deb
 cd /srv/www/repos/apt/debian
-reprepro includedeb buster /srv/src/peach-oled/target/debian/peach-oled_0.1.0_amd64.deb`
+reprepro includedeb buster /srv/src/peach-oled/target/debian/peach-oled_0.1.0_amd64.deb
+```
 
 ## building for aarch64
-`cd /srv/src/peach-oled
+```cd /srv/src/peach-oled
 cargo build --release --target=aarch64-unknown-linux-gnu
-CARGO_TARGET_AARCH64_UNKNOWN_LINUX_GNU_LINKER=/usr/bin/aarch64-linux-gnu-gcc cargo-deb --release --target=aarch64-unknown-linux-gnu`
\ No newline at end of file
+CARGO_TARGET_AARCH64_UNKNOWN_LINUX_GNU_LINKER=/usr/bin/aarch64-linux-gnu-gcc cargo-deb --release --target=aarch64-unknown-linux-gnu
```

Comment:
This is looking great @mhfowler :)

Two small notes:

1) the aarch64 linker variable needs to be defined before the `cargo build` command, otherwise it will fail (ie. the linker path is required by both commands: `cargo build` and `cargo deb`)

2) `cargo-deb` should be `cargo deb`

Reply:
P.s. I actually just added `.cargo/config` to all the microservice repos. This removes the need for the linker variable to be exported, since it is defined in the `config` file and automatically detected by both `cargo build` and `cargo deb`.

I figure it's better to have these variables coded in-place, rather than requiring the extra export step each time.

Reply:
nice I pulled the repos with the new .cargo/config 
and it worked!

going to try seeing if I can get the package installed to the pi from the debain repo :)

Reply:
successfully installed peach-oled to the pi via apt-get (so nice)

but peach-oled getting some errors when running.. will look into it a bit 
```
Nov 11 13:53:06 peach systemd[1]: /lib/systemd/system/peach-oled.service:12: Address prefix is invalid, ignoring assignment: ~127.0.0.1
Nov 11 13:53:06 peach systemd[1]: /lib/systemd/system/peach-oled.service:18: Unknown lvalue 'ProtectDevices' in section 'Service', ignoring
Nov 11 13:53:06 peach systemd[1]: /lib/systemd/system/peach-oled.service:25: Failed to parse namespace type string, ignoring: CLONE_NEWUSER
```